Source/ablastr/math/fft/WrapRocFFT.cpp

```diff
@@ -66,6 +66,69 @@ namespace ablastr::math::anyfft
         return fft_plan;
     }
 
+    FFTplan CreatePlanMany(int * real_size, amrex::Real * real_array,
+                           Complex * complex_array, const direction dir, const int dim,
+                           int howmany, int * inembed, int istride, int idist,
+                           int * onembed, int ostride, int odist)
+    {
+        FFTplan fft_plan;
+
+        const std::size_t lengths[] = {AMREX_D_DECL(std::size_t(real_size[0]),
+                                                    std::size_t(real_size[1]),
+                                                    std::size_t(real_size[2]))};
+
+        const std::size_t roc_istride[] = {AMREX_D_DECL( (dir == direction::R2C) ? std::size_t(istride * real_size[0]) : std::size_t(istride * (real_size[0]/2+1)),
+                                                          std::size_t(istride * real_size[1]),
+                                                          std::size_t(istride * real_size[2]))};
+
+        const std::size_t roc_ostride[] = {AMREX_D_DECL( (dir == direction::R2C) ? std::size_t(ostride * (real_size[0]/2+1)) : std::size_t(ostride * real_size[0]),
+                                                          std::size_t(ostride*real_size[1]),
+                                                          std::size_t(ostride*real_size[2]))};
+
+        rocfft_plan_description desc = nullptr;
+        rocfft_status result = rocfft_plan_description_set_data_layout(
```

Comment:
The documentation for rocFFT seems to indicate that `rocfft_plan_description_create` should be called first.

Reply:
No need of the plan description anymore. 
However, the rocFFT wrapper is currently less flexible than that of FFTW and cuFFT.